SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
```

Comment:
Looks like this is a comment from a template file no?

Reply:
🤦‍♂️

Reply:
Replaced by 0db678b8bfd5d7d2db2672ed7692fcc5f026e320